content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -125,8 +125,7 @@ driver. These attributes are specific to each driver and not
 standardized. See the documentation of each CSI driver for further
 instructions.
 
-As a cluster administrator, you can use a [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) to control which CSI drivers can be used in a Pod, specified with the
-[`allowedCSIDrivers` field](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicyspec-v1beta1-policy).
+As a cluster administrator, you can use a [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) to control which CSI drivers can be used in a Pod, specified with the {{< api-reference page="policies-resources/pod-security-policy-v1beta1" anchor="AllowedCSIDriver" text="`allowedCSIDrivers` field">}} structure.
```

Comment:
Hi @feloy ,
I don't fully understand the use of `anchor` vs. `text`.
Should you set both parameters together or do you only set one or the other?

Reply:
```
<a href="{{ $page.URL }}{{if $anchorArg}}#{{ $anchorArg }}{{end}}">
{{if $textArg}}
  {{ $textArg }}
{{else if $anchorArg}}
  {{ $anchorArg }}
{{else}}
  {{ $metadata.kind }}
{{end}}
</a> 
```

Text is used to force the text displayed into the link.

If text is not defined, the displayed text will be the name of the Resource, or, if anchor is defined, the value of anchor.

Reply:
Yes, okay. So, any text can be entered into the shortcode for both parameters.
The text should be consistent and normalized?